### ISSUE TITLE
[expo-av][android] cannot read assets embedded by packager in apk `res` directory

### ISSUE THREAD

**Issue by jonathanstiansen** [expo/expo#8888]
### Summary of Issue (just a few sentences)
When building a new APK with bare-workflow the build process does not bundle assets into the apk. 
Then when we turn off the ability for Expo updates to work, it can't even load assets. Which seems like an odd thing to break. Though I understand that the OTA is just a JS bundle, that expo-asset loads. I'm not sure it should also block everything else though.

### Environment - output of `expo diagnostics` & the platform(s) you're targeting
```  
Expo CLI 3.20.9 environment info:
    System:
      OS: macOS 10.15.4
      Shell: 5.7.1 - /bin/zsh
    Binaries:
      Node: 13.14.0 - ~/.nvm/versions/node/v13.14.0/bin/node
      Yarn: 1.22.4 - /usr/local/bin/yarn
      npm: 6.14.4 - ~/.nvm/versions/node/v13.14.0/bin/npm
      Watchman: 4.9.0 - /usr/local/bin/watchman
    IDEs:
      Xcode: 11.4.1/11E503a - /usr/bin/xcodebuild
    npmPackages:
      expo: ~37.0.3 => 37.0.12 
      react: ~16.9.0 => 16.9.0 
      react-dom: ~16.9.0 => 16.9.0 
      react-native: ~0.61.5 => 0.61.5 
      react-native-web: ~0.11.7 => 0.11.7 
    npmGlobalPackages:
      expo-cli: 3.21.9
```

### Reproducible Demo
I've built a minimal Repo to reproduce this. [Here](https://github.com/jonathanstiansen/bareWorkflorUpdatesRepo) it is.

### Steps to Reproduce
Essentially:

1. clone repo
2. run `yarn install`
3. run `yarn android --variant release`
4. (optional) Confirm updates are not working by changing some text in `App.js` and running `expo publish`
5. Notice that there is no sound played on startup.
6. Go into the `app/build.gradle` file and change `UPDATES_ENABLED` flag to be true
7. run `yarn android --variant release` 
8. You'll hear the sound work again.

### Expected Behavior vs Actual Behavior
According to these guides https://docs.expo.io/guides/offline-support/#bundle-your-assets-inside-your-standalone-binary. In expo-updates readme [here](https://www.npmjs.com/package/expo-updates#appbuildgradle) it says:

> Add the following Gradle build script. This provides expo-updates with some important metadata about the update and assets that are embedded in your APK.

Saying that assets should be embedded in the APK.

We had this working with expo-updates 1.2.

We found some code in the android version that changed in expo-updates 1.3 for how it bundles assets, that's the best we've got.

**Comment by brentvatne** [comment]
hello! can you explain what you mean by expo-updates 1.2? the latest version is 0.2.9

**Comment by jonathanstiansen** [comment]
@brentvatne Hey brent, sorry I was sloppy. I meant 0.1.2, and 0.1.3. 

You'll note the demo app has the latest version, 0.2.9.

**Comment by esamelson** [comment]
Hey @jonathanstiansen - thanks for the report and the repro. It looks like this is actually an issue with `expo-av` not knowing how to read assets bundled into an apk's `res` directory. (You would see the same issue if you used `expo-av` in a bare React Native app without `expo-updates` -- it just so happens that enabling `expo-updates` is a workaround for the issue as it DOES know how to read assets from the `res` folder and copies them to a different location.)

I'm going to rename this issue to track it appropriately. We'll try and have a fix out soon but can't promise it on any given timeline.

**Comment by jonathanstiansen** [comment]
Sounds good. We aren’t able to use that workaround given ur constraints but we will eagerly await a fix :-) thank you for your great work guys!

**Comment by jonathanstiansen** [comment]
@esamelson just wondering if this has made it into a release? And if not, how do I keep track for myself without having to bug you?

**Comment by esamelson** [comment]
Hey @jonathanstiansen -- I've just published `expo-av@8.3.0` (tagged as `next` on NPM) with this commit included. Apologies it took so long; generally updates to individual packages between SDK releases are haphazard, though we'd like to publish more frequently! For now, you can check the [package CHANGELOG](https://github.com/expo/expo/blob/master/packages/expo-av/CHANGELOG.md) to determine how recently it's been published.

**Comment by cruzach** [comment]
8.3.0 is now `latest` for expo-av, so I'm going to close this issue. @jonathanstiansen if you're still seeing this problem after integrating the fix, let us know
